x/lockup/keeper/lock.go

```diff
@@ -313,6 +345,22 @@ func (k Keeper) addTokensToLock(ctx sdk.Context, lock *types.PeriodLock, coins s
 	return nil
 }
 
+func (k Keeper) removeTokensFromLock(ctx sdk.Context, lock *types.PeriodLock, coins sdk.Coins) error {
```

Comment:
Lets add a comment here, and some descriptions of where its ok to be called from?

Reply:
Okay